bonusopdracht.js

```diff
@@ -0,0 +1,53 @@
+// 1. Maak drie knoppen op de pagina: Sorteer op prijs, AmbiLight TV's en Uitverkochte exemplaren. Gebruik de code die je in opdracht 1b, 1c en 1d hebt gemaakt en schrijf dit om naar functies zodat je ze kunt aanroepen op het moment dat de buttons geklikt worden. Zorg ervoor dat de functies de uitkomsten in de console loggen als de gebruiker op de bijbehorende knop klikt. Tip: lees hiervoor paragraaf 7.4 op EdHub eens door!
+
+
+// Filteren op uitverkochte tv's:
+
+function filterSoldOutTvs() {
+    const soldOut = inventory.filter((tv) => {
+        return tv.originalStock === tv.sold;
+    })
+    console.log(soldOut);
+    createTvList(soldOut)
+}
+
+// Filteren op AmbiLight:
+
+function filterAmbiLightTvs() {
+    const hasAmbiLight = inventory.filter((tv) => {
+        return tv.options.ambiLight === true
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
        return tv.options.ambiLight
```